src/bridge/bridge.c

```diff
@@ -574,6 +586,24 @@ run_bridge (const gchar *interactive,
 }
 
 static void
+print_rules (gboolean opt_privileged)
+{
+  CockpitRouter *router = NULL;
+  GList *bridges = NULL;
+  CockpitTransport *transport = cockpit_interact_transport_new (0, 1, "--");
+
+  router = setup_router (transport, opt_privileged, &bridges);
+
+  cockpit_router_dump_rules (router);
+
+  g_object_unref (router);
+  g_list_free (bridges);
+  g_object_unref (transport);
+  cockpit_packages_free (packages);
+  packages = NULL;
```

Comment:
I don't want to get into a bunch of refactoring here, packages is used in several places. Marius is likely to need to changes this all soon anyways.

I can move the free to the end of the file if that makes things better for now.